### PR TITLE
Fixup K8s operator generated code 

### PR DESCRIPTION
**Pull request by astoycos**
Fix an error where our generated register and clients wasn't getting updated, use go package names rather than absolute paths to manage all of this, NOTE this code must be stored under your gopath

### CODE REVIEW

File: bpfman-operator/Makefile [hunk 1]
```diff
@@ -181,10 +181,7 @@ ifeq ($(VERIFY_CODEGEN), true)
 VERIFY_FLAG=--verify-only
 endif
 
-OUTPUT_PKG ?= $(shell pwd)/bpfman-operator/pkg/client
-APIS_PKG ?= $(shell pwd)/bpfman-operator
-CLIENTSET_NAME ?= versioned
-CLIENTSET_PKG_NAME ?= clientset
+PKG ?= github.com/bpfman/bpfman/bpfman-operator
 COMMON_FLAGS ?= ${VERIFY_FLAG} --go-header-file $(shell pwd)/hack/boilerplate.go.txt
```
Comment (Billy99) [inline bpfman-operator/Makefile:185]:
```suggestion
COMMON_FLAGS ?= ${VERIFY_FLAG} --go-header-file ${PKG}hack/boilerplate.go.txt
```
Does this need to change too? My output from sample command (see `--go-header-file`):
```
/home/bmcfall/go/src/github.com/bpfman/bpfman/bpfman-operator/bin/client-gen \
	--clientset-name "versioned" \
	--input-base "" \
	--input "github.com/bpfman/bpfman/bpfman-operator/apis/v1alpha1" \
	--output-package "github.com/bpfman/bpfman/bpfman-operator/pkg/client/clientset" \
	 --go-header-file /home/bmcfall/go/src/github.com/bpfman/bpfman/bpfman-operator/hack/boilerplate.go.txt
```

Comment (Billy99) [reply bpfman-operator/Makefile:185]: Never-mind, it fails to build with that change.

**Comment by astoycos** [comment]
@dave-tucker You're going to hate this but I cannot figure out a way past this and I'd rather have this hack then not have coverage.

I left a long code comment explaining all of the issues.

**Review by Billy99 — APPROVED** [review]
LGTM

**Review by anfredette — COMMENTED** [review]
lgtm

**Pull request merged**